content/python/concepts/built-in-functions/terms/super/super.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'super()'
+Description: 'implicitly refer to the superclass'
+Subjects:
+  - 'Computer Science'
```

Comment:
```suggestion
  - 'Computer Science'
  - 'Data Science'
```

Since Python is commonly used in data science.

Reply:
I've added Data Science as a tag. I've actually never encountered super() as a data science term that why I didn't put it in, but I admit, I haven't taken Codecademy's Data Science course.

Reply:
Yeah these are relatively loose associations meant to bolster the entry's SEO.